diff --git a/plugin/pbcopy.vim b/plugin/pbcopy.vim
index 3bb622d..cc9bd9e 100644
--- ./plugin/pbcopy.vim
+++ ./plugin/pbcopy.vim
@@ -1,7 +1,6 @@
 let g:vim_pbcopy_local_cmd = "pbcopy"
 
-vnoremap <silent> cy :<C-U>call <SID>copyVisualSelection(visualmode(), 1)<CR>
-nnoremap <silent> cy :set opfunc=<SID>copyVisualSelection<CR>g@
+vnoremap <silent> y :<C-U>call <SID>copyVisualSelection(visualmode(), 1)<CR>
 
 function! s:getVisualSelection()
     let [lnum1, col1] = getpos("'<")[1:2]
